tests/compile_tests/audio_mem_test/sndopen_mem_test.bas

```diff
@@ -0,0 +1,1311 @@
+$Console:Only
+Option _Explicit
+Option _ExplicitArray
+
+Dim buffer As String: buffer = LoadTestData
+Print "Size ="; Len(buffer)
+
+
+Dim h As Long: h = _SndOpen(buffer, "memory")
```

Comment:
Thank you for adding this test, I was wondering how this worked :) This is pretty handy, it's probably just as good as being able to pass a `_MEM` in.